parser/csv_parser/LinksOnlyCsvParser.py

```diff
@@ -116,7 +116,8 @@ def main_loop(self):
             # pepPos2 - if pepPos2 is not set fill list with default value (-1)
             # ToDo: might need changing for xiUI where pepPos is not optional
             self.logger.info(id_item['abspos2'])
-            if id_item['abspos2'] == -1 or math.isnan(id_item['abspos2']):
+            abs2 = id_item['abspos2']
+            if abs2 == -1: #or math.isnan(abs2):
```

Comment:
can it not be `nan`?

Reply:
i'm not sure, evidently i thought it couldn't... maybe need to check this